src/sections/Community/Meshmates/index.js

```diff
@@ -0,0 +1,155 @@
+                                    <td className="feature"> 
+                                        <h4> Run Meshery </h4>
+                                        <p> Put on your user hat and walk-through all of Meshery’s features and functions as a user. </p>
+                                    </td>
+                                </tr>
+                                <tr> 
+                                    <td className="icon"><img alt="icon" src={c_icon} /></td>
+                                    <td className="feature"> 
+                                        <h4> Build Meshery </h4>
+                                        <p> Confirm that you have a usable development environment. </p>
+                                    </td>
+                                </tr>
+                                <tr> 
+                                    <td className="icon"><img alt="icon" src={c_icon} /></td>
+                                    <td className="feature"> 
+                                        <h4> Contribute </h4>
+                                        <p> Grab an open issue or suggest a new one. </p>
+                                    </td>
+                                </tr>
+                            </table>
+                        </Col>
+                        <Col lg={6}>
+                            <div className="callout-card">
+                                <img className="card-img" alt="community" src={hero} />
+                                <div className="card-text">
+                                    <h4> Community Call </h4>
+                                    <p> Layer5 hosts official monthly community calls where users and contributors can discuss about any topic and demonstrate  use-cases. </p>
+                                    <h4 className="highlight"> Interested? </h4>
+                                    <p> You can register below for the next Community Call. </p>
+                                    <Row>
+                                        <input type="text" placeholder="Your Email Address" />
+                                        <Button secondary title="Subscribe" />
+                                    </Row>
+                                    <p className="unsubscribe"> You can unsubscribe any time. No spam. </p>
+                                </div>
+                            </div>
+                        </Col>
+                    </Row>
+                </div>
+                <div className="conduct">
+                    <h2> Code of Conduct </h2>
+                    <p> The comfort and safety of Layer5 community members is our priority. You must agree to the Code of Conduct to participate in the Layer5 community, and any violations of the Code of Conduct will be taken seriously. <br/>
+                    To report any violations please fill out this incident form. </p>
+                    <Button primary title="Fill Form"></Button>
```

Comment:
Link to form - https://docs.google.com/forms/d/e/1FAIpQLSeWzC5HjlHugFjB0TtaAVnSkPPqsRQ3JRYjdwyDXf0oyRxcdQ/viewform